lib/widgets/compose_box.dart

```diff
@@ -22,6 +22,59 @@ import 'store.dart';
 import 'text.dart';
```

Comment:
commit message nit:

```
compose: Compose-box border is subtle in dark mode 
```

Let's [start the commit summary line with an imperative verb](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-summary-part-2) instead.